tailcall/server.graphql

```diff
@@ -9,7 +9,16 @@ schema
   )
 
   # Specify a base url for all HTTP requests.
-  @upstream(timeout: 5) {
+  @upstream(timeout: 5)
+
+  @telemetry(
+    export: {
+      otlp: {
+        url: "http://localhost:4318"
```

Comment:
the otel collector should be able to receive telemetry via http on port 4318

https://github.com/kevinmichaelchen/temporal-saga-grpc/blob/main/docker-compose.yml#L39